src/Castle.Core/Castle.Core.csproj

```diff
@@ -4,7 +4,16 @@
 
 	<PropertyGroup>
 		<TargetFrameworks>net45;netstandard2.0;netstandard2.1</TargetFrameworks>
-	</PropertyGroup>
+	
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
+    <IncludeSymbols>true</IncludeSymbols>
+    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
```

Comment:
Please move these properties to the following location (and make sure to use tabs for indentation, not spaces):

https://github.com/castleproject/Core/blob/a7454e5d43e29fee1451a0190b333d23b47497f1/buildscripts/common.props#L27